package.json

```diff
@@ -40,7 +40,7 @@
     "@octokit/rest": "^16.36.0",
     "hbs": "^4.1.0",
     "parse-diff": "^0.6.0",
-    "probot": "^9.9.0"
+    "probot": "file:../probot"
```

Comment:
This seems unintentional 😛

Reply:
It's a WIP pull request :D You can replace it with `probot/probot#upgrade-to-octokit-rest-17` if you want to test yourself